### ISSUE TITLE
Rename pocket_ related strings to be non-pocket specific.

### ISSUE THREAD

**Issue by k88hudson** [mozilla/activity-stream#2902]
Ideally it would be good to rename pocket-related strings to top-sections etc. However, we would like to do this without forcing new translations – perhaps there might be a way to bulk update all languages?

**Comment by Mardak** [comment]
I would also think some developers would love to rename "string_name5" back to just "string_name" after a number of iterations, but typically they just live on in the code as they are.

@flodolo if we were to do some bulk renaming, we would do that for each of the lines in https://github.com/mozilla/activity-stream-l10n/tree/master/l10n ?

**Comment by flodolo** [comment]
You can do bulk renaming in the repository, and Pontoon will pick up the changes. 

Note that we lose some metadata in the process in Pontoon (history, attribution), potentially pending suggestions, so it works but it's not ideal.

Which strings are we talking about? There are 6 or 7 with pocket in the ID, but several of them actually refer to Pocket.

**Comment by Mardak** [comment]
Looks like #2901 had this commit: https://github.com/mozilla/activity-stream/commit/7e285c50288ea36588fe590ee46dacb114215c13#diff-906742e1af7418eb557312dc9217f2f1R139

It wanted to add/rename from:
```ini
pocket_read_more=Popular Topics:
pocket_read_even_more=View More Stories
```
to:
```ini
topics=Popular Topics:
topics_read_more=View More Stories
```

**Comment by k88hudson** [comment]
Yeah, I think it's fine to live with it if we have to, @csadilek just brought it up as part of the refactor and figured it might cause confusion

**Comment by flodolo** [comment]
I've checked and we're going to lose at least some suggestions. If we can avoid renaming existing strings in the repo, I would say it's much safer.

There might be also a risk of some creative translations referencing Pocket (I doubt it).

Having said that, if we want to rename them, we still can (you also need to fix the localization comments).
